backend/inc/fnc.css.php

```diff
@@ -938,7 +938,7 @@ function insert_cssrelation( $idcss, $idcssfile ) {
 	$sql  = 'INSERT INTO ' . $cms_db['css_upl'] . ' (idcss, idupl) VALUES ';
 	$sql .= '(' . $idcss . ', ' . $idcssfile . ')';
 	$db->query($sql);
-	return mysqli_insert_id($this->db->Link_ID);
+	return mysqli_insert_id($GLOBALS['db']->Link_ID);
```

Comment:
Same here: Why not using `$db->Link_ID`?

Reply:
Same problem. $this->db is not accessible everywhere.